pipeline.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.1.1
```

Comment:
Should this be incremented everytime who touches this?

Reply:
No, I added like in documentation sample, the documentation of https://git.autodesk.com/Quantum/CILibrary I mean.